tests/daemon/unit/stores/test_basestore.py

```diff
@@ -1,14 +1,18 @@
+import pathlib
 import uuid
 
 import pytest
 
+from daemon.models import DaemonID
+from daemon.models.base import StoreItem
 from daemon.stores.base import BaseStore
 from jina.helper import random_identity, random_uuid
+from daemon import __root_workspace__
 
-keys = [uuid.UUID(random_identity()) for _ in range(3)]
+keys = [DaemonID(f'jflow-{uuid.UUID(random_identity())}') for _ in range(3)]
 
 
-store_items = {keys[0]: {'object': 'abc'}, keys[1]: {'object': 'hij'}, keys[2]: {}}
+store_items = {keys[0]: StoreItem(), keys[1]: StoreItem(), keys[2]: StoreItem()}
 
 
 def test_base_store_clear():
```

Comment:
`test_base_store_clear` & `test_base_store_del` don't make sense anymore. Can you update/delete them?